spiceaidocs/docs/local-acceleration/index.md

```diff
@@ -23,6 +23,29 @@ Data Storage: Ensure that the local storage has enough capacity to store the acc
 
 Data Security: Assess data sensitivity and secure network connections between edge and data connector when replicating data for further usage. Assess the security of any Data Accelerator that is external to the Spice runtime and connected to the Spice runtime. Implement encryption, access controls, and secure protocols.
 
+## Data Refresh
+
+Dataset acceleration can be configured in `full` (the entire dataset is refreshed) or `append` (new data from a dataset source is appended) acceleration modes.
+
+The data refresh interval for an accelerated dataset in `full` mode can be specified via [refresh_interval](/reference/spicepod/datasets#accelerationrefresh_interval) or triggered via Api (`POST`, `/v1/datasets/:name/refresh`).
```

Comment:
I would not conflate these concepts. Separate into separate paragraphs.

Reply:
Updated - thank you!

--
![image](https://github.com/spiceai/docs/assets/981580/887d8414-a4ed-4b14-90f5-432ab84c1028)